mdgru/data/grid_collection.py

```diff
@@ -281,13 +282,13 @@ def save(self, data, filename, tporigin=None):
             except Exception as e:
                 ending = os.path.splitext(self.featurefiles[0])[-1]
         if ending in ['.mhd']:
-            skio.imsave(filename + ending, data, plugin='simpleitk')
+            skio.imwrite(filename + ending, data, plugin='simpleitk')
```

Comment:
If I'm not mistaken, for skimage.io the method call would still be imsave. Otherwise, the PR looks fine! Thanks a lot for submitting it. When that change has been fixed, I will merge it.